tests/test_other.py

```diff
@@ -11171,6 +11171,10 @@ def test_unistd_seek(self):
   def test_unistd_mkdir(self):
     self.do_run_in_out_file_test('wasmfs/wasmfs_mkdir.c')
 
+  @with_wasmfs
```

Comment:
It doesn't need to be in this PR, but I had a comment in another (previous? not sure) PR about renaming this to `also_with_wasmfs` for consistency.

Reply:
Sorry, I thought I committed it. Will add it in now.